modules/dists/BlockDist.chpl

```diff
@@ -348,157 +344,38 @@ record Block {
     this.rank = rank;
     this.idxType = idxType;
     this.sparseLayoutType = _to_unmanaged(sparseLayoutType);
-    this._pid = if _isPrivatized(value) then _newPrivatizedClass(value) else nullPid;
-    this._instance = value;
+    this.chpl_distHelp = new chpl_PrivatizedDistHelper(
+                          if _isPrivatized(value) then _newPrivatizedClass(value) else nullPid,
+                                                       value);
   }
 
     proc init(_pid : int, _instance, _unowned : bool) {
       this.rank = _instance.rank;
       this.idxType = _instance.idxType;
       this.sparseLayoutType = _instance.sparseLayoutType;
-      this._pid      = _pid;
-      this._instance = _instance;
-      this._unowned  = _unowned;
+      this.chpl_distHelp = new chpl_PrivatizedDistHelper(_pid,
+                                                         _instance,
+                                                         _unowned);
     }
 
     proc init(value) {
       this.rank = value.rank;
       this.idxType = value.idxType;
       this.sparseLayoutType = value.sparseLayoutType;
-      this._pid = if _isPrivatized(value) then _newPrivatizedClass(value) else nullPid;
-      this._instance = _to_unmanaged(value);
+      this.chpl_distHelp = new chpl_PrivatizedDistHelper(
+                                                         if _isPrivatized(value) then _newPrivatizedClass(value) else nullPid,
+                                                         _to_unmanaged(value));
```

Comment:
Nit, but it looks like this indentation is significantly different than that in the initializer in 347-349. I definitely prefer the syntax from 347-349 (and don't think this one fits within the character limit).

Reply:
Good point, I'll change all of these to be more like:
```chapel
...new blah(
     if blah
       then blah
       else blah,
     nextArg);
```